lib/double_entry/errors.rb

```diff
@@ -9,6 +9,7 @@ class TransferCodeTooLongError < RuntimeError; end
   class DuplicateAccount < RuntimeError; end
   class DuplicateTransfer < RuntimeError; end
   class AccountWouldBeSentNegative < RuntimeError; end
+  class AccountWouldBeSentPositiveError < RuntimeError; end
```

Comment:
none of the other error's names end in `Error`. I think we should be consistent here